secretmanager/snippets/regional_samples/delete_regional_secret.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+"""
+Command line application and sample code for deleting an existing regional
+secret.
+"""
+
+import argparse
+
+
+# [START secretmanager_v1_delete_regional_secret]
+def delete_regional_secret(project_id: str, location_id: str, secret_id: str) -> None:
+    """
+    Deletes the secret with the given name and all of its versions.
+    """
+
+    # Import the Secret Manager client library.
+    from google.cloud import secretmanager_v1
+
+    # Endpoint to call the regional secret manager sever
+    api_endpoint = f"secretmanager.{location_id}.rep.googleapis.com"
+
+    # Create the Secret Manager client.
+    client = secretmanager_v1.SecretManagerServiceClient(
+        client_options={"api_endpoint": api_endpoint},
+    )
+
+    # Build the resource name of the secret.
+    name = f"projects/{project_id}/locations/{location_id}/secrets/{secret_id}"
+
+    # Delete the secret.
+    client.delete_secret(request={"name": name})
```

Comment:
Just double checking - fine to not return the response?

Reply:
Its alright if it doesnt return a value